src/helpers/formatCreationDate.js

```diff
@@ -1,17 +1,11 @@
-export default function formatCreationDate(totalMinutes) {
-	const hours = Math.floor(totalMinutes / 60);
-	const minutes = totalMinutes % 60;
-	let result = '';
+function formatCreationDate() {
```

Comment:
1. Let's use AF
2. Usually, helpers (utils) do not have default export since there are many functions and there could be only single default export